account_bank_statement_import_txt_xlsx/models/account_bank_statement_import_sheet_mapping.py

```diff
@@ -12,6 +12,13 @@ class AccountBankStatementImportSheetMapping(models.Model):
     name = fields.Char(
         required=True,
     )
+
+    skipped_rows = fields.Integer(
```

Comment:
`rows_to_skip_start`? What if someone would add _end later

Reply:
I'd go for a more precise `header_offset`